front_end/handlers/DevelopmentLoginHandler.py

```diff
@@ -0,0 +1,37 @@
+from helper import *
+from tornado.web import *
+import traceback
+from content import *
+
+
+settings_dict = load_yaml_dict(read_file("/Settings.yaml"))
+content = Content(settings_dict)
```

Comment:
Create the content object within the get and post functions rather than defining outside the class.